tembo-cli/src/cli/sqlx_utils.rs

```diff
@@ -0,0 +1,48 @@
+use crate::Result;
+use base64::{engine::general_purpose, Engine as _};
+use spinners::Spinner;
+use spinners::Spinners;
+use sqlx::migrate::Migrator;
+use sqlx::Pool;
+use sqlx::Postgres;
+use std::path::Path;
+use temboclient::models::ConnectionInfo;
```

Comment:
Shoot - small issue. We want to use the data plane API instead of the control plane API for the username / password info because we are planning to remove from control plane the database passwords so that we don't need to save them. I.e. we are planning for control plane API to not return password / username information.

Reply:
Ah yes, thanks for the reminder. You had mentioned that when I was implementing the TF provider. I will change it.

Reply:
you'll want to use this one https://api.data-1.use1.tembo.io/swagger-ui/#

Reply:
Do you mind if I merge this PR and replace the call in the next one?